test/OpenTelemetry.Exporter.OneCollector.Benchmarks/Program.cs

```diff
@@ -0,0 +1,46 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using BenchmarkDotNet.Running;
+
+namespace OpenTelemetry.Exporter.OneCollector.Benchmarks;
+
+internal static class Program
+{
+    public static void Main(string[] args)
+    {
+        if (Debugger.IsAttached)
```

Comment:
Just double check if this condition is expected here.

Reply:
I left it in because it is kind of nice trick to help test/debug the benchmark code when you are working on it.

Reply:
Added a note/comment in there explaining the block.